debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/antlr/listener/RenameTableParserListener.java

```diff
@@ -28,6 +33,14 @@ public RenameTableParserListener(MySqlAntlrDdlParser parser) {
     public void enterRenameTableClause(MySqlParser.RenameTableClauseContext ctx) {
         TableId oldTable = parser.parseQualifiedTableId(ctx.tableName(0).fullId());
         TableId newTable = parser.parseQualifiedTableId(ctx.tableName(1).fullId());
+        if (parser.getTableFilter().isIncluded(oldTable) && !parser.getTableFilter().isIncluded(newTable)) {
+            LOG.warn("Renaming whitelisted table {} to non-whitelisted table {}, this can lead to schema inconsistency", oldTable, newTable);
```

Comment:
What I'm wondering, should we rather throw a hard error in this case, stopping the connector? It might be better than having to recover from this inconsistent state. OTOH it'd mean to use the preview whitelist config change support. WDYT?

Reply:
@gunnarmorling Please revistit the updated PR. Now the schema change tool will mostly work so I'd keep it as `WARN` right now. When https://issues.jboss.org/browse/DBZ-1330 will be implemented the `WARN` should be changed to `ERROR` in case of `stored.only.monitored.tables` set to `true`.